simtools/simtel/simtel_config_writer.py

```diff
@@ -365,11 +365,8 @@ def _write_array_triggers_file(self, array_triggers, model_path, telescope_model
             trigger_dict = self._get_array_triggers_for_telescope_type(array_triggers, tel_type)
             trigger_lines[tel_type] = f"Trigger {trigger_dict['multiplicity']['value']} of "
             trigger_lines[tel_type] += ", ".join(map(str, tel_list))
-            if all(trigger_dict["width"][key] is not None for key in ["value", "unit"]):
-                width = trigger_dict["width"]["value"] * u.Unit(trigger_dict["width"]["unit"]).to(
-                    "ns"
-                )
-                trigger_lines[tel_type] += f" width {width}"
+            width = trigger_dict["width"]["value"] * u.Unit(trigger_dict["width"]["unit"]).to("ns")
```

Comment:
A unit will also be provided always, right? If it is empty you would get `dimensionless` instead I assume.

Reply:
The json schema requires an astropy_unit (and with PR #1201 even an astropy unit of time)